base/abstractarray.jl

```diff
@@ -968,6 +968,22 @@ _unsafe_ind2sub(sz, i) = (@_inline_meta; _ind2sub(sz, i))
     setindex!(A, X, inds...)
 
 Store values from array `X` within some subset of `A` as specified by `inds`.
```

Comment:
While you are at it; append something like
```
The syntax `A[inds...] = X` (lowers|is equivalent) to `setindex!(A, X, inds...)`
```
?
Perhaps also change the signature up top to
```
    setindex!(A, X, inds...)
    A[inds...] = X

Store values ...
```